category/category.go

```diff
@@ -162,7 +163,7 @@ func (m *GormRepository) LoadAllRelationshipsOfCategory(ctx context.Context, cat
 		log.Error(ctx, map[string]interface{}{
 			"category_id": categoryID,
 		}, "category not found")
```

Comment:
@kwk I removed the log message because the function `LoadCategoryFromDB` already returns a log message if an error occurs -> 	b21a4be